### PR TITLE
My simple portfolio 1

### PR DESCRIPTION
**Pull request by monicah79**
- Used the nav tag
- Added my social networks
- Followed figma design

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,14 +9,14 @@
...
             <a href="">My Logo</a>
             <img src="Images/Menu-icon.svg" alt="">
-        </header>
+        </nav>
         <section class="homepage-section">
             
         <h1>
-        I'm Monicah <br>
+        I'm Monicah   <br>
         Glad to see you!</h1>
```
Comment (franclobo) [inline index.html:20]: - [ ] **[OPTIONAL]** Please 🙏🏽 use the correct font styles to be similar to the Figma template: 👇🏽 

image


| Figma template | Your design |
|---|---|
| image | image |

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @monicah79, 

# Required Changes ♻️

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- No linter errors ✔️ 
- Professional README file ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._

# **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

# **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by monicah79** [comment]
hey quick question so what am i supposed to update right now coz the pull
request i had earlier had already closed it

**Comment by monicah79** [comment]
have done the changes so does it mean i dont have to open a new pull
request i just send the same link after pushing

**Review by frank1738 — COMMENTED** [review]
Hi <STUDENT_MENTION>, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You submitted your project in the wrong milestone. Currently, you are working on `Portfolio: setup and mobile first` milestone but you submitted it in ` Portfolio: finish mobile version`.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by monicah79** [comment]
ahahaha believe me this projects are crazy i didnt even notice,,,hehehehe
sorry about that,,,thanks for the review

**Comment by monicah79** [comment]
can i merge it now since on my dashboard its saying submit

**Comment by monicah79** [comment]
Can i merge it now or ,,,since there is no option on request another pull
request

**Comment by monicah79** [comment]
Awesome, thanks!

**Comment by franclobo** [comment]
I'm sorry, my misstake. There weren't required changes just one optional. I approved your project @monicah79

**Comment by monicah79** [comment]
No problem, thanks!

**Pull request merged**